app/api.py

```
@@ -0,0 +1,72 @@
+import logging
```

Ruff (0.11.9)

`logging` imported but unused

Remove unused import: `logging`

(F401)